apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -116,18 +116,69 @@ export default class CoreLibrary {
     });
   }
 
+  /**
+   * Draws bubbles for each variable in the `variableBubbles` array. Labels are truncated with an ellipsis
+   * if they exceed the maximum character limit. Values are truncated as needed to fit the remaining space.
+   *
+   * @param {Object[]} this.variableBubbles - An array of objects, each representing a variable to be displayed. Each object should include:
+   *  - `name`: A string identifier for the variable used to get the value from the JSInterpreter.
+   *  - `label`: A label for the variable to be displayed in the bubble.
+   *  - `location`: An object specifying the `x` and `y` coordinates where the bubble should be drawn.
+   */
   drawVariableBubbles() {
+    const config = {
```

Comment:
I was tired of having these magic numbers repeated in multiple places. Thoughts? Should this be stored elsewhere, like in a constants file as VARIABLE_BUBBLES_CONFIG?